R/InMemoryAnnData.R

```diff
@@ -118,6 +120,25 @@ InMemoryAnnData <- R6::R6Class("InMemoryAnnData", # nolint
         private$.var_names <- private$.validate_obsvar_names(value, "var")
         self
       }
+    },
+    #' @field obsm
+    obsm = function(value) {
+      if (missing(value)) {
+        private$.obsm
+      } else {
+        # TODO: validate obsm
+        private$.obsm <- private$.validate_array_collection_generic(value, "obsm", c(self$n_obs()), expected_rownames=rownames(self))
+        self
+      }
+    },
+    varm = function(value) {
+      if (missing(value)) {
+        private$.varm
```

Comment:
Add trackstatus comment

```suggestion
        # trackstatus: class=InMemoryAnnData, feature=get_varm, status=done
        private$.varm
```